src/contracts/pods/EigenPod.sol

```diff
@@ -222,6 +232,9 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
 
         emit ValidatorRestaked(validatorIndex);
 
+        //record validator's new restaked balance
+        _validatorPubkeyHashToInfo[validatorPubkeyHash].restakedBalanceGwei = REQUIRED_BALANCE_GWEI;
```

Comment:
why don't we just get the effective balance for the initial proof. will make it cheaper and result in the same outcome. will only allow restaking if their balance is >31.75 ETH. pinging @ChaoticWalrus here too.

Reply:
Yeah fair enough